functions/Get-DbaBackupHistory.ps1

```diff
@@ -303,13 +293,18 @@ Lots of detailed information for all databases on sqlserver2014a and sql2016.
 						$where = "$where $wherearray"
 					}
 					
-					$sql = "$select $from $where ORDER BY backupset.backup_finish_date DESC"
+					$sql = "$select $from $where ORDER BY backupset.Backup_Finish_Date DESC"
 				}
 				
 				if (!$last)
 				{
 					Write-Debug $sql
-					$sourceserver.ConnectionContext.ExecuteWithResults($sql).Tables.Rows | Select-Object * -ExcludeProperty BackupSetRank, RowError, Rowstate, table, itemarray, haserrors
+					$results = $sourceserver.ConnectionContext.ExecuteWithResults($sql).Tables.Rows | Select-Object * -ExcludeProperty BackupSetRank, RowError, Rowstate, table, itemarray, haserrors
+					$results = $results | select-object *, @{Name="Fullname";expression={$_.Path}}
+					foreach ($result in $results)
+					{ 
+						$result | Select-DefaultField -property * -excludeproperty Fullname

```

Comment:
I tried to make it as easy as possible, so you dont need to explicitly set inclusions